extensions/httpserver/description.yml

```
@@ -0,0 +1,30 @@
+extension:
+  name: httpserver
+  description: Simple DuckDB HTTP API extension
+  version: 1.0.0
```

```suggestion
  version: 0.1.0
```
Maybe, given it's experimental?